images/template-placeholders/single-product.svg

```diff
@@ -0,0 +1 @@
+<svg width="893" height="451" fill="none" xmlns="http://www.w3.org/2000/svg"><rect width="451" height="451" rx="2" fill="#E5E5E5"/><rect x="470" width="423" height="71" rx="2" fill="#E5E5E5"/><rect x="470" y="147" width="423" height="17" rx="2" fill="#E5E5E5"/><rect x="470" y="172" width="423" height="17" rx="2" fill="#E5E5E5"/><rect x="470" y="305" width="140" height="53" rx="2" fill="#E5E5E5"/><rect x="470" y="217" width="83" height="18" rx="2" fill="#E5E5E5"/><path d="M481.599 96.876a1 1 0 0 1 1.802 0l2.48 5.162a.998.998 0 0 0 .768.558l5.676.764a1 1 0 0 1 .557 1.714l-4.143 3.954a1 1 0 0 0-.293.903l1.028 5.634a1 1 0 0 1-1.459 1.06l-5.04-2.719a1 1 0 0 0-.95 0l-5.04 2.719a1 1 0 0 1-1.459-1.06l1.028-5.634a1 1 0 0 0-.293-.903l-4.143-3.954a1 1 0 0 1 .557-1.714l5.676-.764a.998.998 0 0 0 .768-.558l2.48-5.162ZM506.599 96.876a1 1 0 0 1 1.802 0l2.48 5.162a.998.998 0 0 0 .768.558l5.676.764a1 1 0 0 1 .557 1.714l-4.143 3.954a1 1 0 0 0-.293.903l1.028 5.634a1 1 0 0 1-1.459 1.06l-5.04-2.719a1 1 0 0 0-.95 0l-5.04 2.719a1 1 0 0 1-1.459-1.06l1.028-5.634a1 1 0 0 0-.293-.903l-4.143-3.954a1 1 0 0 1 .557-1.714l5.676-.764a.998.998 0 0 0 .768-.558l2.48-5.162ZM531.599 96.876a1 1 0 0 1 1.802 0l2.48 5.162a.998.998 0 0 0 .768.558l5.676.764a1 1 0 0 1 .557 1.714l-4.143 3.954a1 1 0 0 0-.293.903l1.028 5.634a1 1 0 0 1-1.459 1.06l-5.04-2.719a1 1 0 0 0-.95 0l-5.04 2.719a1 1 0 0 1-1.459-1.06l1.028-5.634a1 1 0 0 0-.293-.903l-4.143-3.954a1 1 0 0 1 .557-1.714l5.676-.764a.998.998 0 0 0 .768-.558l2.48-5.162ZM556.599 96.876a1 1 0 0 1 1.802 0l2.48 5.162a.998.998 0 0 0 .768.558l5.676.764a1 1 0 0 1 .557 1.714l-4.143 3.954a1 1 0 0 0-.293.903l1.028 5.634a1 1 0 0 1-1.459 1.06l-5.04-2.719a1 1 0 0 0-.95 0l-5.04 2.719a1 1 0 0 1-1.459-1.06l1.028-5.634a1 1 0 0 0-.293-.903l-4.143-3.954a1 1 0 0 1 .557-1.714l5.676-.764a.998.998 0 0 0 .768-.558l2.48-5.162ZM581.599 96.876a1 1 0 0 1 1.802 0l2.48 5.162a.998.998 0 0 0 .768.558l5.676.764a1 1 0 0 1 .557 1.714l-4.143 3.954a1 1 0 0 0-.293.903l1.028 5.634a1 1 0 0 1-1.459 1.06l-5.04-2.719a1 1 0 0 0-.95 0l-5.04 2.719a1 1 0 0 1-1.459-1.06l1.028-5.634a1 1 0 0 0-.293-.903l-4.143-3.954a1 1 0 0 1 .557-1.714l5.676-.764a.998.998 0 0 0 .768-.558l2.48-5.162Z" fill="#E5E5E5"/></svg>
```

Comment:
We'll need to replace this with the latest SVG provided by design. OK to leave it as is for now though and update it together with the Archive one in https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/5053.